truss/templates/server/common/errors.py

```diff
@@ -171,3 +173,42 @@ def intercept_exceptions(
             "Internal Server Error", exc_info=filter_traceback(model_file_name)
         )
         raise UserCodeError(str(exc))
+
+
+def _loc_to_dot_sep(loc: Tuple[Union[str, int], ...]) -> str:
+    # From https://docs.pydantic.dev/latest/errors/errors/#customize-error-messages.
+    # E.g. ('items', 1, 'value') -> items[1].value.
```

Comment:
nit: maybe would be helpful to have a couple examples in the comment, with both  x: str and : int

Reply:
The example itself contains both str and int already. But I added an explaining comment (and changed string concat to a signle `join`).